mabl snippets/SetValueOnReactInputElement.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+/**
+ * This javascript snippet sets a value on an input for applications using react.  mabl can generally support setting the input
+ * value directly by interacting with the page but this function provides a template for setting it via a javascript step, if needed
```

Comment:
This seems generic to me, and perhaps the check for the `getOwnPropertyDescriptor` bit could just be merged with our other click snippet?  Hadn't thought of that before.

Reply:
yeah, could be worth pulling into one generic thing.  To be fair, I didn't think of it either, just hunted down how react-testing library was simulating input events and found the linked react issue/comments.